phoenix-client/dependency-reduced-pom.xml

```diff
@@ -0,0 +1,263 @@
+<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
Is this used anymore? I could not find any references, if so we can get rid of it.